lib/node_modules/@stdlib/lapack/base/README.md

```diff
@@ -30,13 +30,13 @@ Many LAPACK routines work with banded matrices, which are stored compactly in tw
 
 ### General Band Matrix Storage
 
-A general band matrix of size `M`-by-`N` with `KL` subdiagonals and `KU` superdiagonals is stored in a two-dimensional array `AB` with `KL+KU+1` rows and `N` columns.
+A general band matrix of size `M`-by-`N` with `KL` subdiagonals and `KU` superdiagonals is stored in a two-dimensional array `A` with `KL+KU+1` rows and `N` columns.
```

Comment:
@aayush0325 What was the rationale for changing `A` to `AB` here and below? Now at, e.g., L101, you have

```
Element `A[i, j]` is stored in `A[1+i-j, j]`.
```

which no longer makes sense.